src/tests/ftest/util/security_test_base.py

```diff
@@ -25,9 +25,32 @@
 import os
 import random
 from general_utils import pcmd
+from apricot import TestWithServers
+from general_utils import create_local_file
```

Comment:
(pylint-no-name-in-module) No name 'create_local_file' in module 'general_utils'
(pylint-unused-import) Unused create_local_file imported from general_utils
(pylint-ungrouped-imports) Imports from package general_utils are not grouped